circuit_maintenance_parser/parsers/zayo.py

```diff
@@ -24,6 +43,14 @@ def parse_html(self, soup):
         self.parse_bs(soup.find_all("b"), data)
         self.parse_tables(soup.find_all("table"), data)
 
+        if data:
```

Comment:
just wondering why do we need this check?

Reply:
For the case where we might decide that there's no relevant content in the email (and hence return an empty `data` dict) I wouldn't want to just fill in these additional fields in an otherwise empty dict.

Reply:
I see, otherwise you could just rely on Pydantic validation once the `Maintenance` output is instantiated.